sky/clouds/service_catalog/aws_catalog.py

```diff
@@ -94,6 +99,17 @@ def get_vcpus_from_instance_type(instance_type: str) -> Optional[float]:
     return common.get_vcpus_from_instance_type_impl(_df, instance_type)
 
 
+def get_default_instance_type(cpu: Optional[str] = None) -> Optional[str]:
+    if cpu is None:
+        cpu = str(_DEFAULT_NUM_VCPUS)
+    # The metal instance is not included in the default instance family.
+    instance_type_prefix = f'{_DEFAULT_INSTANCE_FAMILY}.'
+    instance_type_suffix = 'xlarge'
+    df = _df[_df['InstanceType'].str.startswith(instance_type_prefix) &
+             _df['InstanceType'].str.endswith(instance_type_suffix)]
```

Comment:
Like above: why not just filter by the prefix + '.'?

Reply:
Good idea. Fixed.